packages/conjure-client/src/fetchBridge/__tests__/fetchBridgeTests.ts

```diff
@@ -482,7 +483,7 @@ function createFetchRequest(opts: ICreateFetchRequestOpts): RequestInit {
         headers: {
             ...headers,
             Authorization: `Bearer ${token}`,
-            "Fetch-User-Agent": "foo/1.2.3",
+            "Fetch-User-Agent": "foo/1.2.3 conjure-client/0.0.0",
```

Comment:
Will this test fail on a tag build?

Reply:
🤦

Reply:
what do you think about the name? I wasn't super happy with it, but its the name of the npm package

Reply:
From an observability perspective the name of this repo could be nice (as we do with both dialogue and cjr) but I don’t have a strong preference.